configapi/api_inventory.go

```diff
@@ -1,231 +1,231 @@
 // SPDX-License-Identifier: Apache-2.0
-// Copyright 2024 Canonical Ltd.
+// SPDX-FileCopyrightText: 2024 Canonical Ltd
 
 package configapi
 
 import (
-    "errors"
-    "fmt"
-    "encoding/json"
-    "strings"
-    "net/http"
-
-    "github.com/gin-gonic/gin"
-    "github.com/omec-project/util/httpwrapper"
-    "github.com/omec-project/webconsole/backend/logger"
-    "github.com/omec-project/webconsole/configmodels"
-    "github.com/omec-project/webconsole/dbadapter"
-    "go.mongodb.org/mongo-driver/bson"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/gin-gonic/gin"
+	"github.com/omec-project/util/httpwrapper"
+	"github.com/omec-project/webconsole/backend/logger"
+	"github.com/omec-project/webconsole/configmodels"
+	"github.com/omec-project/webconsole/dbadapter"
+	"go.mongodb.org/mongo-driver/bson"
 )
 
 const (
-    gnbDataColl = "webconsoleData.snapshots.gnbData"
-    upfDataColl = "webconsoleData.snapshots.upfData"
+	gnbDataColl = "webconsoleData.snapshots.gnbData"
+	upfDataColl = "webconsoleData.snapshots.upfData"
 )
 
 func setInventoryCorsHeader(c *gin.Context) {
-    c.Writer.Header().Set("Access-Control-Allow-Origin", "*")
-    c.Writer.Header().Set("Access-Control-Allow-Credentials", "true")
-    c.Writer.Header().Set("Access-Control-Allow-Headers", "Content-Type")
-    c.Writer.Header().Set("Access-Control-Allow-Methods", "POST, GET, DELETE")
+	c.Writer.Header().Set("Access-Control-Allow-Origin", "*")
+	c.Writer.Header().Set("Access-Control-Allow-Credentials", "true")
+	c.Writer.Header().Set("Access-Control-Allow-Headers", "Content-Type")
+	c.Writer.Header().Set("Access-Control-Allow-Methods", "POST, GET, DELETE")
 }
 
 func GetGnbs(c *gin.Context) {
-    setInventoryCorsHeader(c)
-    logger.WebUILog.Infoln("Get all gNBs")
-
-    var gnbs []*configmodels.Gnb
-    gnbs = make([]*configmodels.Gnb, 0)
-    rawGnbs, errGetMany := dbadapter.CommonDBClient.RestfulAPIGetMany(gnbDataColl, bson.M{})
-    if errGetMany != nil {
-        logger.DbLog.Errorln(errGetMany)
-        c.JSON(http.StatusInternalServerError, gnbs)
-    }
-
-    for _, rawGnb := range rawGnbs {
-        var gnbData configmodels.Gnb
-        err := json.Unmarshal(mapToByte(rawGnb), &gnbData)
-        if err != nil {
-            logger.DbLog.Errorf("Could not unmarshall gNB %v", rawGnb)
-        }
-        gnbs = append(gnbs, &gnbData)
-    }
-    c.JSON(http.StatusOK, gnbs)
+	setInventoryCorsHeader(c)
+	logger.WebUILog.Infoln("Get all gNBs")
+
+	var gnbs []*configmodels.Gnb
+	gnbs = make([]*configmodels.Gnb, 0)
+	rawGnbs, errGetMany := dbadapter.CommonDBClient.RestfulAPIGetMany(gnbDataColl, bson.M{})
+	if errGetMany != nil {
+		logger.DbLog.Errorln(errGetMany)
+		c.JSON(http.StatusInternalServerError, gnbs)
+	}
+
+	for _, rawGnb := range rawGnbs {
+		var gnbData configmodels.Gnb
+		err := json.Unmarshal(configmodels.MapToByte(rawGnb), &gnbData)
+		if err != nil {
+			logger.DbLog.Errorf("Could not unmarshall gNB %v", rawGnb)
```

Comment:
Do you mind making this update as part of your PR?
```suggestion
			logger.DbLog.Errorf("could not unmarshal gNB %v", rawGnb)
```